tests/runtime/test_grpc_server.py

```diff
@@ -835,7 +835,7 @@ def test_load_model_badmodel_error_response(runtime_grpc_server):
             modelKey="baz",
         )
         stub.loadModel(load_model_request)
-    assert context.value.code() == grpc.StatusCode.INTERNAL
+    assert context.value.code() == grpc.StatusCode.NOT_FOUND
```

Comment:
Shouldn't these have been `NOT_FOUND` from the beginning? Maybe I'm missing some context

Reply:
Nope, you're probably right!